enable_debug_image: true
downsample_factor: 1.0
